sim/src/main/scala/spinal/sim/IVerilogBackend.scala

```diff
@@ -0,0 +1,171 @@
+package spinal.sim
+
+import org.apache.commons.io.FileUtils
+import spinal.sim.vpi.SharedMemIface
+
+import java.io.{File, PrintWriter}
+import java.nio.file.{Files, Paths}
+import scala.sys.process.Process
+
+
+/* README first!
```

Comment:
I think it should be turned into:

```scala
@deprecated("This backend ...")
```

So that it appears on user side

Reply:
Ahh, for this one, we should discuss it with @Dolu1990 

For me, IVerilogBackend works well. Why does it not work?

Reply:
We realy need a working backend for IVerilog, @numero-744 what is the replacement ?